### PR TITLE
Add "What Makes SecureDrop Unique" guide

### LINKED ISSUE #469: What makes SecureDrop unique?
Seeing how SecureDrop has at least one competitor, it would be good to create a document explaining in layman's terms what makes the platform unique. This could either be a standalone readme or a part of the FPF website, or both.

### PR DESCRIPTION
**Pull request by redshiftzero**
## Status

Ready for review

## Description of Changes

Adds the "What Makes SecureDrop Unique" guide written by @trevortimm during last week's docs sprint. Closes #469. This is also the kind of information we should be adding to update securedrop.org's FAQ.

## Testing

Double check any spelling/grammar/typesetting issues. @conorsch and I have both reviewed the content, but feel free to make any comments.

## Checklist

- [x] Docs linting passes locally

### CODE REVIEW

File: docs/what_makes_securedrop_unique.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+has the power to contest the order or refuse to comply if they so wish.
+
+Limits the metadata trail as much as possible
+---------------------------------------------
+
+In many leak cases, the metadata of a journalist's communications—where you’re
+located, who you’re talking to, when you’re talking to them, and how often—can
+lead to trouble just as much as the actual content of your conversations.
+
+With SecureDrop, even if a government serves a court order directly to a news
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: Would look/sound nicer as:

_Even if a government serves a court order directly to a news organization to compel the disclosure of information, SecureDrop logs much less information than email providers or phone companies do._

File: docs/what_makes_securedrop_unique.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+organization to compel the disclosure of information, SecureDrop logs much less
+information than email providers or phone companies do.
+
+The source can only log into SecureDrop through the Tor Browser, which masks the
+source’s IP address to begin with, so there is no indication who the source is
+(unless they disclose it) and where they are sending information from. The Tor IP
+address, the computer and the browser type that the source is using is not logged
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: Oxford comma time?

"The Tor IP address, the computer **,** and the browser type ..."

File: docs/what_makes_securedrop_unique.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+either.
+
+For each source, only the time and date of each submission is logged on the
+server. When a source sends a new message the time and date of the last message
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: Insert comma after message:

"When a source sends a new message **,** the time and date ..."

File: docs/what_makes_securedrop_unique.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+In addition, sources cannot create a custom username that could reveal information
+about them. Instead, SecureDrop automatically generates two random codenames, one
+to show to the source and another to the journalists using the system.
+
+Encrypted and air-gapped
+------------------------
+
+Communications through SecureDrop are both encrypted in transit, so messages cannot
+be easily intercepted and read while they are traversing the internet and are also
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: Capitalize "internet".

File: docs/what_makes_securedrop_unique.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+Protecting against hackers
+--------------------------
+
+A 2014 study showed that 20 of the top 25 news organization had, at one time or
+another, had their networks compromised by state sponsored hackers.
+
+Because of this threat, SecureDrop completely segments its traffic from a news
+organization’s normal network. Submissions are accessed and downloaded using the
+Tails operating system, which boots off of a USB, does not touch your computer’s
+hard drive, and routes all its internet traffic through Tor.
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: Capitalize "internet."

File: docs/what_makes_securedrop_unique.rst [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+In addition, the decryption key for SecureDrop submissions sits on an air-gapped
+computer (not connected to the Internet). This air-gapped computer is the only
+place SecureDrop submissions are decrypted and read so that they are much harder
+for an attacker to access.
+
+Protecting against hackers
+--------------------------
+
+A 2014 study showed that 20 of the top 25 news organization had, at one time or
+another, had their networks compromised by state sponsored hackers.
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: Remove "had" after "another,". Already preceded this clause with "had,"

File: docs/what_makes_securedrop_unique.rst [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+Submissions are decrypted on an air-gapped computer also using Tails. This
+mitigates against the risk that an attacker could send malware through SecureDrop
+in an attempt to infect the news organization’s normal network as well.
+
+The SecureDrop servers also undergo significant system hardening in order to
+make it as difficult as possible for hackers to break in.
+
+So SecureDrop protects sources against networks that are already compromised and
+also protects a news organization’s normal network from attacks that could
+potentially come through SecureDrop.
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: From line 88 to 93, I'd rewrite as:

_The SecureDrop servers also undergo significant system hardening in order to make it as difficult as possible for hackers to break in. By doing so, SecureDrop protects sources against networks that are already compromised, as well as a news organization’s normal network from attacks that could potentially come through SecureDrop._

File: docs/what_makes_securedrop_unique.rst [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+Free and open source software
+-----------------------------
+
+100% of SecureDrop’s code is free and open source. Not only does this mean anyone
+can install SecureDrop themselves, but the code is available online for security
+experts to test for vulnerabilities.
+
+SecureDrop has gone through at least four audits by third party penetration
```
Comment (justintroutman) [inline docs/what_makes_securedrop_unique.rst]: Remove "at least", as we should be more concrete with the number of audits we've had.

**Comment by redshiftzero** [comment]
Thanks for your detailed review @justintroutman! I've made the suggested edits. Let me know if there are any other comments.

File: docs/what_makes_securedrop_unique.rst [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+More recently, there have been a record number of leak prosecutions largely because
+the government has learned they don’t need reporters to testify against their
+sources anymore. Instead, they can just secretly subpoena third party services
+like Google or AT&T or Verizon or Facebook and get a treasure trove of digital
+information on reporters and sources’ communications. For example, the Associated
+Press had twenty of their phone lines subpoenaed without their knowledge in order
+to identify a source. The government also got a warrant for Fox News reporter James
+Rosen’s Gmail account without him knowing. In both cases, their alleged sources
+were prosecuted, even though journalists never directly divulged their sources.
```
Comment (NSkelsey) [inline docs/what_makes_securedrop_unique.rst]: Hi this is a great summary. I would suggest adding dates to examples contained in here. Starting with "More recently," means that in 18th months this paragraph may be less punctual.

even though "the" journalists never

**Pull request merged**